src/main/java/org/softc/armoryexpansion/common/integration/aelib/plugins/constructsarmory/material/ArmorMaterial.java

```diff
@@ -0,0 +1,97 @@
+package org.softc.armoryexpansion.common.integration.aelib.plugins.constructsarmory.material;
+
+import c4.conarm.lib.materials.ArmorMaterialType;
+import c4.conarm.lib.materials.CoreMaterialStats;
+import c4.conarm.lib.materials.PlatesMaterialStats;
+import c4.conarm.lib.materials.TrimMaterialStats;
+import org.softc.armoryexpansion.client.integration.aelib.plugins.tinkers_construct.material.MaterialRenderType;
+import org.softc.armoryexpansion.common.integration.aelib.config.MaterialConfigOptions;
+import org.softc.armoryexpansion.common.integration.aelib.plugins.general.material.BasicMaterial;
+import slimeknights.tconstruct.library.TinkerRegistry;
+
+public class ArmorMaterial extends BasicMaterial implements IArmorMaterial{
+    protected CoreMaterialStats coreMaterialStats;
+    protected PlatesMaterialStats platesMaterialStats;
+    protected TrimMaterialStats trimMaterialStats;
+
+    ArmorMaterial(String identifier, int color) {
+        super(identifier, color, MaterialRenderType.DEFAULT);
+    }
+
+    public ArmorMaterial(String identifier, int color, MaterialRenderType type, CoreMaterialStats coreMaterialStats, PlatesMaterialStats platesMaterialStats, TrimMaterialStats trimMaterialStats) {
+        super(identifier, color, type);
+        this.coreMaterialStats = coreMaterialStats;
+        this.platesMaterialStats = platesMaterialStats;
+        this.trimMaterialStats = trimMaterialStats;
+    }
+
+    @Override
+    public CoreMaterialStats getCoreMaterialStats() {
+        return this.coreMaterialStats;
+    }
+
+    @Override
+    public PlatesMaterialStats getPlatesMaterialStats() {
+        return this.platesMaterialStats;
+    }
+
+    @Override
+    public TrimMaterialStats getTrimMaterialStats() {
+        return this.trimMaterialStats;
+    }
+
+    @Override
+    public boolean isToolMaterial() {
+        return false;
+    }
+
+    @Override
+    public boolean isArmorMaterial() {
+        return null != this.coreMaterialStats
+                || null != this.platesMaterialStats
+                || null != this.trimMaterialStats;
+    }
+
+    @Override
+    public boolean isRangedMaterial() {
+        return false;
+    }
+
+    @Override
+    public boolean registerTinkersMaterialStats(MaterialConfigOptions properties){
+        if (properties.materialEnabled() && properties.armorEnabled()) {
+            slimeknights.tconstruct.library.materials.Material material = TinkerRegistry.getMaterial(this.getIdentifier());
+//            if (!"unknown".equals(material.getIdentifier())) {
+                this.registerArmorStats(material, properties);
+                return true;
+//            }
+        }
+        return false;
+    }
+
+    void registerArmorStats(slimeknights.tconstruct.library.materials.Material material, MaterialConfigOptions properties){
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/YaibaToKen/Armory-Expansion/pullRequest?prid=3896610)